src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    <p className="Person__age">
+      {person.age !== undefined ? `I am ${person.age}` : ''}
+    </p>
```

Comment:
You need to fix conditional rendering everywhere
```suggestion
   {person.age && (
      <p className="Person__age">......</p>
   )}
```